src/Reducer/watch-reducer.js

```diff
@@ -0,0 +1,10 @@
+export const watchReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_TO_WATCHLATER":
```

Comment:
make reducer action type as contants in seperate file and import them it will be a good practise